components/tls/tls-mpc/src/leader.rs

```diff
@@ -104,7 +110,10 @@ impl MpcTlsLeader {
             prf,
             encrypter,
             decrypter,
-            committed: VecDeque::new(),
+            notifier: BackendNotifier::new(),
+            is_decrypting: true,
```

Comment:
isnt `is_decrypting` just a reflection of the `notifier`'s state? Here they are set in the opposite states. Doesnt seem like they should be.